bin/run-browser-test-env

```diff
@@ -12,45 +12,73 @@ if [ -f ".secrets" ]; then
   . .secrets
 fi
 
+ci="no"
```

Comment:
style, optional: Google general style eschews abbreviations especially if they're not core to the system, as it tends to result in problems for the benefit of typing less.

I'd recommend spelling this out, especially internal to the file.

Reply:
done